src/tools/rados/rados.cc

```diff
@@ -1909,7 +1909,7 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
 
   std::string run_name;
   std::string prefix;
-  bool forcefull = false;
+  bool force_full = false;
```

Comment:
Leave this alone

Reply:
Ok.

Note that this was a carry over from somewhere else where I incorrectly changed it and someone told me this was the correct intent.

Reply:
Hmm, I don't have insight into any other PRs.  If there was precedent elsewhere I don't think it _hurts_ to change it and we can leave it in, of course along with the associated instances below.

Reply:
https://github.com/ceph/ceph/pull/46824#discussion_r906158594